packages/expo/src/Updates/Updates.ts

```diff
@@ -12,21 +41,26 @@ export async function reloadFromCache(): Promise<void> {
   await ExponentUpdates.reloadFromCache();
 }
 
-export async function checkForUpdateAsync(): Promise<Object> {
+export async function checkForUpdateAsync(): Promise<UpdateCheckResult> {
   if (!ExponentUpdates.checkForUpdateAsync) {
     throw new UnavailabilityError('Updates', 'checkForUpdateAsync');
   }
   const result = await ExponentUpdates.checkForUpdateAsync();
-  let returnObj: any = {
-    isAvailable: !!result,
-  };
-  if (result) {
-    returnObj.manifest = typeof result === 'string' ? JSON.parse(result) : result;
-  }
-  return returnObj;
+
+  return !!result && typeof result === 'string'
```

Comment:
```suggestion
  return result && typeof result === 'string'
```

we don't need to make it into a boolean here

Reply:
or just `return typeof result === 'string'` to match `fetchUpdateAsync` - it seems we don't need to guard against empty strings here

Reply:
Okay, done.